stac_pydantic/shared.py

```diff
@@ -118,10 +132,10 @@ class StacCommonMetadata(StacBaseModel):
 
     title: Optional[str] = Field(None, alias="title")
     description: Optional[str] = Field(None, alias="description")
-    start_datetime: Optional[datetime] = Field(None, alias="start_datetime")
-    end_datetime: Optional[datetime] = Field(None, alias="end_datetime")
-    created: Optional[datetime] = Field(None, alias="created")
-    updated: Optional[datetime] = Field(None, alias="updated")
+    start_datetime: Optional[UtcDatetime] = Field(None, alias="start_datetime")
+    end_datetime: Optional[UtcDatetime] = Field(None, alias="end_datetime")
+    created: Optional[UtcDatetime] = Field(None, alias="created")
```

Comment:
we can remove the unnecessary `alias` there I believe (not sure why we use it before)